nix/cardano-services/deployments/handle-provider-deployment.yaml

```diff
@@ -0,0 +1,99 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: handle-provider
+    network: preview
+    release: dev-preview-cardanojs
+  name: dev-preview-cardanojs-handle-provider
+spec:
+  selector:
+    matchLabels:
+      app: handle-provider
+      network: preview
+      release: dev-preview-cardanojs
+  template:
+    metadata:
+      labels:
+        app: handle-provider
+        network: preview
+        release: dev-preview-cardanojs
+    spec:
+      containers:
+        - args:
+            - start-provider-server
+          env:
+            - name: ALLOWED_ORIGINS
+              value: chrome-extension://gafhhkghbfjjkeiendhlofajokpaflmk,chrome-extension://efeiemlfnahiidnjglmehaihacglceia
+            - name: ENABLE_METRICS
+              value: "true"
+            - name: HANDLE_POLICY_IDS
+              value: f0ff48bbb7bbe9d59a40f1ce90e9e9d0ff5002ec48f232b49ca0fb9a
+            - name: HANDLE_PROVIDER_SERVER_URL
+              value: https://preview.api.handle.me/
+            - name: LOGGER_MIN_SEVERITY
+              value: debug
+            - name: NETWORK
+              value: preview
+            - name: OGMIOS_SRV_SERVICE_NAME
+              value: dev-preview-cardano-stack.dev-preview.svc.cluster.local
+            - name: POSTGRES_DB
+              value: handle
+            - name: POSTGRES_HOST
+              value: dev-preview-dbsync-db
+            - name: POSTGRES_PASSWORD
+              valueFrom:
+                secretKeyRef:
+                  key: password
+                  name: handle-owner-user.dev-preview-dbsync-db.credentials.postgresql.acid.zalan.do
+            - name: POSTGRES_POOL_MAX
+              value: "10"
+            - name: POSTGRES_PORT
+              value: "5432"
+            - name: POSTGRES_SSL
+              value: "true"
+            - name: POSTGRES_SSL_CA_FILE
+              value: /tls/ca.crt
+            - name: POSTGRES_USER
+              valueFrom:
+                secretKeyRef:
+                  key: username
+                  name: handle-owner-user.dev-preview-dbsync-db.credentials.postgresql.acid.zalan.do
+            - name: SERVICE_NAMES
+              value: handle
+            - name: USE_KORA_LABS
+              value: "true"
+          image: FIXME-ADD-LATEST-IMAGE-REFERENCE
```

Comment:
In slack you mentioned that this PR has already been tested - how was it tested since this should be crashing since there is no overwrite in `deployments.nix`